# Exercises Python Proporsions

Set up a proportion:<br>
$\frac{n1}{d1}=\frac{n2}{d2}$

In [ ]:
n1 = 23
d1 = 4
n2 = 0
d2 = 34
if n2==0:
    answer= n1 * d2 / d1
    print("n2 = ", answer)
if d2==0:
    answer= d1 * n2 / n1
    print("d2 = ", answer)

In [9]:
print(11 - 8)
print(24 + 15)
print(18 / 8)
print(23 * 8)
print((10-9)/3)
print(13/6)
print((33-9)/11)
print(12/11)
print(10/5)

3
39
2.25
184
0.3333333333333333
2.1666666666666665
2.1818181818181817
1.0909090909090908
2.0


## Respostas week 1

x +  8  =  11
Correct! 

-15  + x =  24
Correct! 

8 x =  18
Round your answer to two decimal places.
Correct! 

x/ 8  =  23
Correct! 

3 x +  9  =  10
Round answer to two decimal places
Correct! 

 
x +  4  =  24
Correct! 

-12  + x =  20
Correct! 

6 x =  13
Round your answer to two decimal places.
Try again
The correct answer is  2.17 

x/ 5  =  4
Correct! 

11 x +  9  =  33
Round answer to two decimal places
Correct! 

 
11 x +  -6  =  6
Round answer to two decimal places
Correct! 

5 x +  1  =  11
Round answer to two decimal places
Correct! 

